experiments/tester/package.json

```diff
@@ -16,7 +16,8 @@
     "bundle-dev": "just-scripts bundle --useMetro --bundleName rnTester.dev",
     "start": "react-native start --projectRoot ./src",
     "run-win32": "rex-win32 --bundle RNTester --component RNTesterApp --windowTitle \"FluentUI Tester\" --basePath ./dist --pluginProps",
-    "run-win32-web": "rex-win32 --bundle RNTester --component RNTesterApp --basePath ./dist --useWebDebugger --windowTitle \"FluentUI Tester\" --useLiveReload --pluginProps"
+    "run-win32-web": "rex-win32 --bundle RNTester --component RNTesterApp --basePath ./dist --useWebDebugger --windowTitle \"FluentUI Tester\" --useLiveReload --pluginProps",
+    "run-win32-devmain": "rex-win32 --bundle RNTester --component RNTesterApp --basePath ./dist --pluginProps --useDevMain"
```

Comment:
`--pluginProps` need to be last so that we can pipe extra parameters to it when running `yarn` command.

Reply:
Can  you also add the `windowTitle` parameter?

Reply:
I feel like we shouldn't add an Office-specific command to an open source project, but I understand RNW repo is also doing the same.

Reply:
Is --windowTitle on devmain or the tenant?

Reply:
It shouldn't matter. the local copy of @rex-win32 package supports this prop.

Reply:
Yes, but I needed to make changes to devmain code for this PR to work because I'm developing the native module concurrently.  When I had --windowTitle it wouldn't run with my C++ changes

Reply:
I see. What error did you see? I checked in native change to windowTitle to devmain in this CL: Change 20186530 by REDMOND\kinhln@kinhln16_main on 2020/01/31 10:54:03. It should be available in devmain.

Reply:
Make sure the `--pluginProps` is last.

Reply:
I just did a forward sync with my devmain code so I'll let you know.  I'm not opposed to just checking in the --windowTitle part and just assuming it works, though, tbh, I just needed it not there at the time to test my code.

Reply:
I'll address both of these suggestions in the next push